app/views/collections/show.html.erb

```diff
@@ -114,15 +119,20 @@
               draft: @draft,
               draft_id: @draft.try(:id) }
             } %>
+          <%= render partial: 'shared/loss_report_modal', locals: {
+            options: {
+              provider_id: @provider_id,
+              native_format: native_format,
+              concept_id: @concept_id,
+              revision_id: @revision_id }
+            } %>
+            <!-- Hidden link to allow modal to be shown -->
```

Comment:
Indentation looks off here

Reply:
How would you prefer it? I just copied how the `not_current_provider_modal` locals were indented (above this). It looks right to me..

Reply:
These lines should be on the same indentation as the render partial line.

Reply:
seconded

Reply:
Okay I'll change the other `not_current_provider_modal` block as well